src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -225,12 +251,14 @@ public static partial class ComEventsHelper
         [System.Security.SecurityCriticalAttribute]
         public static System.Delegate Remove(object rcw, System.Guid iid, int dispid, System.Delegate d) { throw null; }
     }
-    public partial class COMException : System.Exception
+    public partial class COMException : ExternalException
     {
         public COMException() { }
+        protected COMException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public COMException(string message) { }
         public COMException(string message, System.Exception inner) { }
         public COMException(string message, int errorCode) { }
+        public override string ToString() { throw null; }
```

Comment:
Looks like this PR and https://github.com/dotnet/corefx/pull/12863 are duplicating some effort and will conflict.